jekyll/_cci2/deployments-and-uploads.md

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
I'm curious about your answer, too @ndintenfass. Wouldn't this mean that our documentation would just infinitely grow? Do we just never delete articles that we rename? I'm mainly worried about directories filling up with dead articles that just redirect.